metadata.rb

```diff
@@ -4,6 +4,7 @@
 license          "Apache 2.0"
 description      "Installs/Configures Microsoft Internet Information Services"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "3.0.0"
+version          "3.0.9"
 supports         "windows"
 depends          "windows", ">= 1.34.6"
+depends          "chef-client", ">= 3.7.0"
```

Comment:
Why the additional dependency? I don't see OpsCode::ChefClient::Helpers or any of the recipes used.